src/quo2/components/buttons/slide_button/view.cljs

```diff
@@ -74,16 +87,19 @@
              {:color colors/white
               :size  20}]]]]]))))
 
-(defn view
+(defn- view-internal
   "Options
   - `on-complete`         Callback called when the sliding is complete
   - `disabled?`           Boolean that disables the button
                           (_and gestures_)
-  - `size`                `:small`/`:large`
+  - `size`                `40`/`48`
```

Comment:
Why moving away from `:small` `:large`? 
Also I would not be passing around `40`/`48`, they look a lot like magic numbers (I am personally against passing keywords as well, I'd `const` everything so there's some safety), but especially with numbers, I'd const them somewhere (withing the component itself makes sense, since if you are using the component, you are requiring that namespace), but that's a larger discussion I take, maybe @J-Son89 @ilmotta @erikseppanen @flexsurfer have an opinion? (apologies if it's been discussed already)

TLDR:
`[slidebutton.view/view {:size 40}]` vs `[slidebutton/view {:size slidebutton/size-small}` (or `slidebutton/size-40`)
(I am for the latter, which is what we do for colors etc)

Reply:
Thanks @cammellos, I believe this is still an open discussion. I agree with you here and like this suggestion the most 👍

Reply:
@cammellos, this is a recurring discussion in PRs.

My opinion has always been that magical design values should be avoided. Tomorrow a designer can change their mind and then we'd need to go through an error-prone process to update all magical values. Sometimes, these magical values also hurt readability because the developer needs to constantly memorize whatever they mean.

The deeper discussion to me is related to using semantic values versus magical values. For example, we're full of `50` and `60` numbers in the code, colors are hardcoded willy-nilly, shadows too use numbers, and so on. One argument in favor of using magical design values is that they eliminate this translation layer between code and Figma, which can help reduce UI bugs.

The code is also a symptom of the way designers use Figma. I've once integrated with a design system that avoided magical values like the plague, and that in turn positively translated to the code. I don't believe our design team will be able to make such drastic changes in Figma, so realistically, it'll be probably up to us to change the code independently.

No discussion oriented towards achieving consensus has been done in our team, I guess it's about time :)

Reply:
Yep, I think this could do with some finalised decision. There could also be a set of sizes in figma foundations similar to typography etc so using :small :medium are consistent across components and designs.

Reply:
I also would vote for using keywords/constants instead of numbers. More up for keywords (`:size-40`) to have a finite list of options when using within `cond`.

cc @ulisesmac, since we had a similar discussion [regarding size for context tags component](https://github.com/status-im/status-mobile/pull/16962#issuecomment-1677542905)

Reply:
I'll set up a vote in the mobile discord and we can record the decision

Reply:
Updated to use: `:size/s-40` and `:size/s-48`